next-cloudinary/src/lib/cloudinary.ts

```diff
@@ -116,7 +117,7 @@ export function getPublicId(src) {
  * createPlaceholderUrl
  */
 
-export function createPlaceholderUrl({ src, placeholder = true, config }) {
+export function createPlaceholderUrl({ src, placeholder = true, config }: { src?: any, placeholder?: boolean | string, config?: any }) {
```

Comment:
Should `src` be typed as `string` or maybe `URL`?

Reply:
That's a fair comment. Might be worth creating specific issues to tackle specific areas. @colbyfayock what do you think?